Assets/MixedRealityToolkit/Extensions/CameraExtensions.cs

```diff
@@ -36,10 +37,73 @@ public static bool IsInFOV(this Camera camera, Vector3 position)
             Vector3 screenPoint = camera.WorldToViewportPoint(position);
 
             return screenPoint.z >= camera.nearClipPlane && screenPoint.z <= camera.farClipPlane
-                && screenPoint.x >= 0 && screenPoint.x <= 1 
+                && screenPoint.x >= 0 && screenPoint.x <= 1
                 && screenPoint.y >= 0 && screenPoint.y <= 1;
         }
 
+
+        // Help to clear caches when new frame runs
```

Comment:
I'm curious to hear opinions on naming for these static variables, as well as whether people have ideas for how to implement this in a better way.

Reply:
I feel like this is complicated enough to be in a utility class by itself?

Other thing that comes up is concurrency. I think....we are fine because Unity restricts all calls to the main thread and this is an extension of Camera. So this should force all calls to IsInFOVConeCached to be serialized and thus not possible for two threads to be accessing the static resources concurrently. Thoughts?

Reply:
I tried to think of a good location and name for this, but couldn't really think of anything. I'm thinking again this might be good to refactor once we have other code that does FOV checks where we want caching.

However, to help clarify things I prefixed the variables with inFOVCone so people understand this is related to the fov cone checks.